src/Uno.UWP/UI/Notifications/IBadgeUpdaterExtension.skia.cs

```diff
@@ -0,0 +1,9 @@
+﻿#nullable enable
+
+namespace Uno.UI.Notifications
+{
+	internal interface IBadgeUpdaterExtension
+    {
+		void SetBadge(string? value);
```

Comment:
If I'm right there is only on macOS where we can set some text, on all other platforms you are parsing the `value` as `int`. Wondering if it would not be just easier / safer to request a `int?` instead?

Reply:
I guess it would be ok, as no Skia target supports string-based value for now and the interface is internal anyway